appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/resource/deployer/JMSDestinationDefinitionDeployer.java

```diff
@@ -43,7 +35,13 @@
 import org.jvnet.hk2.config.TransactionFailure;
 import org.jvnet.hk2.config.types.Property;
 
-import static com.sun.appserv.connectors.internal.api.ConnectorsUtil.deriveResourceName;
+import com.sun.appserv.connectors.internal.api.ConnectorConstants;
```

Comment:
Imports should be ordered by alphabet. Static imports should be first or last, but together, that is ok.

Reply:
Sorry I assumed the ee4j Eclipse formatter took care of the order. I will minimise import statement changes in the future.